django_6pm/django_6pm/templates/index.html

```diff
@@ -0,0 +1,8 @@
+{% if request.user.is_authenticated %}
+<p>{{ request.user.username }} you are logged in. Click <a href="{% url 'logout' %}">Logout</a> to logout</p>
```

Comment:
Only welcome message should be shown like: `Welcome Raiqa` and there should be a simple logout button.

Reply:
I am thinking of adding a profile avatar alongwith a dropdown or something in the navbar but that navbar is in the product views templates. It will be easier to add it once these two branches are merge. Will it be okay if I do it then?